PhysIO/tests/unit/readin/tapas_physio_readin_bids_test.m

```diff
@@ -31,10 +31,10 @@
 % results
 function test_readin_bids_ppu3t(testCase)
 
-% run BIDS PPU example and extract physio
+% Get PhysIO public repo base folder from this file's location
 pathPhysioPublic = fullfile(fileparts(mfilename('fullpath')), '..', '..', '..');
-% TODO: Make generic!
-pathExamples =  fullfile(pathPhysioPublic, '..', 'examples');
+pathExamples = tapas_physio_get_path_examples(pathPhysioPublic);
```

Comment:
These are the tests that I have been working on - they are not complete nor fully functional yet